Add x64 native tools to documentation (for usage)

After `dotnet new` and `dotnet restore` (which appear to work fine, as does `dotnet run`), I get an error doing a native compile:
```
PS C:\users\Public\hello-world> dotnet compile --native
Compiling hello-world for DNXCore,Version=v5.0

Compilation succeeded.
    0 Warning(s)
    0 Error(s)

Time elapsed 00:00:01.8077755
Input Assembly: C:\users\Public\hello-world\bin\Debug\dnxcore50\hello-world.dll
LINK : fatal error LNK1104: cannot open file 'kernel32.lib'
Linking of intermediate files failed.
```

And with `-c Release` as suggested here https://github.com/dotnet/cli/issues/633
```
PS C:\users\Public\hello-world> dotnet compile --native -c Release
Compiling hello-world for DNXCore,Version=v5.0

Compilation succeeded.
    0 Warning(s)
    0 Error(s)

Time elapsed 00:00:01.8150396
Input Assembly: C:\users\Public\hello-world\bin\Release\dnxcore50\hello-world.dll
LINK : fatal error LNK1181: cannot open input file 'kernel32.lib'
Linking of intermediate files failed.
```
Installed just now from `https://dotnetcli.blob.core.windows.net/dotnet/dev/Binaries/Latest/dotnet-win-x64.latest.zip`

Environment:
```
PS C:\users\Public\hello-world> dotnet --version
.NET Command Line Tools (1.0.0)
Runtime Environment:
 OS Name:     Windows
 OS Version:  10.0.10586
 OS Platform: Windows
 Runtime Id:  win10-x64
```

--- a/README.md
+++ b/README.md
@@ -63,7 +63,7 @@ The following command will perform native compilation using the C++ Codegenerato
 
     dotnet build --native --cpp
 
-This will drop a native single binary in `./bin/[configuration]/[framework]/native/[binary name]` that you can run.
+If you are in Windows, make sure that you run the above command inside the *VS 2015 x64 Native Tools* prompt, otherwise you will get errors. This command will drop a native single binary in `./bin/[configuration]/[framework]/native/[binary name]` that you can run.
 
 **Note:** At this point, only the `helloworld` and `dotnetbot` samples will work with native compilation.
 

